.github/workflows/ci.yml

```
@@ -462,72 +498,16 @@ jobs:
     # fake-LLM failures while child services remained alive, so keep it
     # manual until that suite is stable enough to promote.
     timeout-minutes: 25
-    needs: [detect-changes, ci-circuit-breaker-fast, mergeability-gate]
+    needs: [detect-changes, ci-circuit-breaker-linux-runtime, mergeability-gate]
     if: >-
       github.event_name == 'workflow_dispatch' &&
       github.event.inputs.full_windows_e2e == 'true' &&
-      needs.detect-changes.outputs.docs-only != 'true' &&
+      needs.detect-changes.outputs.heavy-code-changed == 'true' &&
       needs.mergeability-gate.outputs.skip-heavy-jobs != 'true'
     runs-on: blacksmith-4vcpu-windows-2025
```

actionlint (1.7.12)

[error] label "blacksmith-4vcpu-windows-2025" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)